src/uk/gov/defra/ffc/Provision.groovy

```diff
@@ -1,16 +1,18 @@
 package uk.gov.defra.ffc
 
 class Provision implements Serializable {
-  static def createResources(ctx, repoName, pr) {
+  static def createResources(ctx, environment, repoName, pr) {
     def filePath = 'provision.azure.yaml'
     if(ctx.fileExists(filePath)) {
-      deletePrResources(ctx, repoName, pr)   
+      deletePrResources(ctx, repoName, pr)
       createAllResources(ctx, filePath, repoName, pr)
     }
+    createPrDatabase(ctx, environment, repoName, pr)
```

Comment:
Should this be within the `if`, checking for the existence of `provision.azure.yaml`?

Reply:
The provision file only lists azure resources, which currently is only queues. The db migration is cloud agnostic so sits outside of that, and will only run.

I think it perhaps looks odd as athe file check is inside the other method. I've refactored slightly to make it clearer.